### ISSUE TITLE
load large data from bigquery to python

### ISSUE THREAD

**Issue by karrey05** [GoogleCloudPlatform/google-cloud-python#3818]
I want to load more than 39 million rows into python and it worked fine a few weeks ago, but now it only returns 100,000 rows. I checked the query in UI and it returned 39 million lines.

```
from google.cloud import bigquery as bq
import google_auth_oauthlib.flow

query = '''select ... from ...'''

bigquery_client = bq.Client()
table = bq.query.QueryResults(query=query,client=bigquery_client)
table.use_legacy_sql = False
table.run()

# transfer bigquery data to pandas dataframe
columns=[field.name for field in table.schema]
rows = table.fetch_data()
data = []
for row in rows:
    data.append(row)
```

I also tried async_query.py, and played with
`rows =query_job.results().fetch_data(max_results=1000000)`
But it seems like they put a cap of 100,000 limit on it somewhere. Is there a way to overwrite the cap? or more efficient way to perform bigquery to python calculation.

**Comment by karrey05** [comment]
I want to load more than 39 million rows into python and it worked fine a few weeks ago, but now it only returns 100,000 rows.  I checked the query in UI and it returned 39 million lines.

```
from google.cloud import bigquery as bq
import google_auth_oauthlib.flow

query = '''select ... from ...'''

bigquery_client = bq.Client()
table = bq.query.QueryResults(query=query,client=bigquery_client)
table.use_legacy_sql = False
table.run()

# transfer bigquery data to pandas dataframe
columns=[field.name for field in table.schema]
rows = table.fetch_data()
data = []
for row in rows:
    data.append(row)
```

I also tried async_query.py, and played with 
`rows =query_job.results().fetch_data(max_results=1000000). `
But it seems like they put a cap of 100,000 limit on it somewhere. Is there a way to overwrite the cap? or more efficient way to perform bigquery to python calculation.

**Comment by karrey05** [comment]
Turns out the version of bigquery client was outdated.  The issue is fixed after upgrading the version to 0.26.0.  I am closing the issue.
